src/main/java/org/gridsuite/study/server/networkmodificationtree/NetworkModificationNodeInfoRepositoryProxy.java

```diff
@@ -43,18 +45,22 @@ public NetworkModificationNodeInfoEntity toEntity(AbstractNode node) {
                                                                                       modificationNode.getLoadFlowStatus(),
                                                                                       StudyService.toEntity(modificationNode.getLoadFlowResult()),
                                                                                       modificationNode.getSecurityAnalysisResultUuid(),
-                                                                                      modificationNode.getBuildStatus());
+                                                                                      modificationNode.getBuildStatus(),
+                                                                                      modificationNode.getModificationsToExclude());
         return completeEntityNodeInfo(node, networkModificationNodeInfoEntity);
     }
 
     @Override
     public NetworkModificationNode toDto(NetworkModificationNodeInfoEntity node) {
+        @SuppressWarnings("unused")
+        int ignoreSize = node.getModificationsToExclude().size();
```

Comment:
It's normal.
Code here is needed to load the lazy collection, and the annotation is used to avoid a warning.
Same thing was done in actions-server to get the countries of a form contingency list (in ContingencyListService.java)

Reply:
Maybe add a comment in the code to explain that ?

Reply:
Comment added